dist/types/models/index.d.ts

```diff
@@ -5,3 +5,4 @@ export { Call } from "./Call";
 export { Account } from "./Account";
 export { Transfer } from "./Transfer";
 export { SystemTokenTransfer } from "./SystemTokenTransfer";
+export { VestingSchedule } from "./VestingSchedule";
```

Comment:
You may want to remove the generated files under dist or ignore that folder when committing

Reply:
Got it.